CMakeLists.txt

```diff
@@ -26,6 +27,49 @@ option(USE_NCCL "Use NCCL library" ON)
 option(USE_SENTENCEPIECE "Download and compile SentencePiece" ON)
 option(USE_STATIC_LIBS "Link statically against non-system libs" OFF)
 option(GENERATE_MARIAN_INSTALL_TARGETS "Generate Marian install targets (requires CMake 3.12+)" OFF)
+option(M32_BINARIES "Generate 32bit binaries even when building outside outside of WASM. Useful for" OFF)
```

Comment:
Unfinished sentence?

Reply:
This change came from @XapaJIaMnu.
@XapaJIaMnu Do you have any preference on what to write here?

Reply:
Oh god, sorry, i have forgotten to finish that. It's supposed to be `Generate 32bit binaries even when building outside of WASM. Useful for testing some WASM specific functionality without the need for the compiling to WASM.` Could you change it?